src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -23,24 +23,63 @@
 
     <section name="What is Javadoc comment">
       <p>
-      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition or field definition.
+        Javadoc comment is a multiline comment <code>/*  */</code> that starts with the <b>*</b> character and placed above the class definition, interface definition, enum definition, method definition or field definition.
+        If an annotation precedes any of the definitions listed above, then the comment should be placed before the annotation.
+        If a number of comments are put sequentially, only the last one will be used.
+      </p>
+      <p>
+        The first sentence has a special meaning and should be clear and punchy, short and ended by a dot symbol.
```

Comment:
please add in front:
`Javadoc comment contains of: summary(first) sentence, documentation section, tag section.`